File: src/Feedback.Analyzer.Backend/Feedback.Analyzer.Application/Organizations/Commands/CreateOrganizationCommand.cs
using Feedback.Analyzer.Api.Models.DTOs;
using Feedback.Analyzer.Domain.Common.Commands;

namespace Feedback.Analyzer.Application.Organizations.Commands;

/// <summary>
/// Represents a command to create a new organization. 
/// </summary>
public class CreateOrganizationCommand : ICommand<OrganizationDto>
{
    /// <summary>
    /// The data required to create a new organization.
    /// </summary>
    public OrganizationDto Organization { get; set; } = default!; 
}

File: src/Insight.Box.FrontEnd/src/modules/organizations/models/CreateOrganizationCommand.ts
import {Command} from "@/infrastructure/models/command/Command";
import type {Organization} from "@/modules/organizations/models/Organization";

/*
 * Represents a command to create an organization.
 */
export class CreateOrganizationCommand extends Command {

   /*
    * Organization to created
    */
    public organization: Organization;

    constructor(organization: Organization)
    {
        super();
        this.organization = organization
        }

}